eclipse-cdt-cloud/timeline-chart
# [lerna] upgrade to recent version

## Description
Upgraded lerna from v4.x to 7.x, getting rid of several big security vulnerabilities, that were present in the older version.


## How to test?
Repo build should be successful and the example application should run as expected.  

The publishing "script" entries for this repo are identical to the ones used in `theia-trace-extension`, and so i have not gone through the process of testing publishing using a local registry. I can do it, if deemed necessary.

## `yarn audit`

Before this PR (master):
![image](https://github.com/eclipse-cdt-cloud/timeline-chart/assets/25749063/c07df046-8628-47fe-bd02-0353d2ed77af)

After:
![image](https://github.com/eclipse-cdt-cloud/timeline-chart/assets/25749063/38850dee-b77a-47b3-b31c-af0f2a68ba44)
